src/main/java/ui/Ui.java

```diff
@@ -106,5 +107,16 @@ public void showHelpList() {
         out.println("3. " + HelpCommand.MESSAGE_USAGE);
         out.println("4. " + AddCommand.MESSAGE_USAGE);
         out.println("5. " + ExitCommand.MESSAGE_USAGE);
+        out.println("6. " + EditCommand.MESSAGE_USAGE);
+    }
+
+    public void showCardContent(Card card) {
```

Comment:
Would it be better to include "edit" inside the method name? (e.g. showCardContentBeforeEdit) as it is only for the edit command